.github/workflows/push.yml

```diff
@@ -17,8 +17,8 @@ jobs:
           distribution: 'temurin'
           java-version: '11'
 
-      - name: Maven Build
-        run: mvn clean spring-javaformat:validate package
```

Comment:
`spring-javaformat:validate` is designed for all code, not only for the generated one.
With this change, the CI will not enforce code updates (non-generated) format.

Reply:
i.e on the test.yml

still this is not good that validate misses formatting on regular code... looking further

Reply:
well, it was named "Maven Build" before you changed it :-) 
This is the first job that is run when a PR is pushed.

Reply:
just to make sure I'm not wrong - on:pull_request should fire the test action . on:push should fire the push action

the test action should fail if we missed the formatting, of all code
the push should add a sanity if we want... not sure about that yet

Reply:
So here is my confusion:
This PR runs when a PR is merged.
I'm happy with changing this to `mvn verify` but we need to make sure `spring-javaformat:validate` is part of parent pom plugin.
I also think that we should preserve `clean` goal here. Not sure about the state of the FS at this point.